token-lending/program/src/instruction.rs

```diff
@@ -283,6 +285,29 @@ pub enum LendingInstruction {
         /// The new owner
         new_owner: Pubkey,
     },
+    // 12
+    /// Make a flash loan.
+    ///
+    ///   0. `[writable]` Destination liquidity token account, minted by reserve liquidity mint.
+    ///   1. `[writable]` Reserve account.
+    ///   2. `[]` Lending market account.
+    ///   3. `[]` Derived lending market authority.
+    ///   4. `[]` Temporary memory
+    ///   5. `[]` Flash Loan Receiver Program Account, which should have a function (which we will
+    ///   call it `ExecuteOperation(amount: u64)` to mimic Aave flash loan) that has tag of 0.
+    ///   6. `[]` Flash Loan Receiver Program Derived Account
+    ///   7. `[]` Token program id
+    ///   8. `[writable]` Host fee receiver.
```

Comment:
Thinking about this more, maybe there shouldn't be a host fee for flash loans, which are made programmatically, versus regular loans, which are often made through a frontend host. @jstarry?

Reply:
@jordansexton It could be nice to keep around since developers might create UI's for rebalancing or bots for arbitrage for others to use and this fee allows them to get compensated for that service

Reply:
I made it mandatory so that we know the number of variable number of accounts for `executeOperation`. I think if the users doesn't want the `host_fee_receiver` they can just sent it to the same `flash_loan_fee_receiver`.

Reply:
Makes sense. I can't think of a better solution so let's run with it :)